saltproc/depcode.py

```diff
@@ -16,14 +17,24 @@ class Depcode(ABC):
     population, active, and inactive cycles. Contains methods to read template
     and output files, and write new input files for the depletion code.
 
+    Attributes
+    -----------
+    param : dict of str to type
+        Holds depletion steo parameter information
+    sim_info : dict of str to type
+        Holds simulation settings information
+    iter_inputfile : str
+        Path to depletion code input file for depletion code rerunning.
+    iter_matfile : str
+        Path to iterative, rewritable material file for depletion code
+        rerunning. This file is modified during  the simulation.
+
     """
 
     def __init__(self,
                  codename,
                  exec_path,
-                 template_inputfile_path,
-                 iter_inputfile,
-                 iter_matfile,
+                 template_inputfiles_path,
```

Comment:
Is this just a variable name? I would suggest adding in an additional underscore in the middle so it doesn't look like you are trying to put two variable names on the same line without a comma.

Reply:
good suggestion